src/languages/en.js

```diff
@@ -108,6 +108,7 @@ export default {
         pay: 'Pay',
         viewDetails: 'View Details',
         settleElsewhere: 'I\'ll settle up elsewhere',
+        decline: 'Decline',
```

Comment:
Should this be in a more generic location? It seems pretty non-specific to IOUs.

Reply:
'Cancel' is common enough that we're using it in multiple cases, but we're not using 'Decline' elsewhere. Let's wait until a label is reused before moving it to common.

Reply:
I think it makes sense to keep strings grouped by use/features until they are reused.